### ISSUE TITLE
Footer Date Text Contrast

### ISSUE THREAD

**Issue by ManasMadan** [DSC-JSS-NOIDA/blogzen#401]
The Date mentioned in footer does not have a appropriate contrast ratio to be clearly visible

![image](https://github.com/DSC-JSS-NOIDA/blogzen/assets/84731321/83d69547-d780-4b8b-804c-04a44a2abb39)

**Comment by ManasMadan** [comment]
I have the required knowledge of the techstack to solve the issue.

Please assign it to Me under GSSoC 23
